build.sbt

```diff
@@ -25,7 +25,6 @@ scalacOptions ++= Seq(
   "-encoding", "UTF-8",       // yes, this is 2 args
   "-feature",
   "-unchecked",
-  "-Xfatal-warnings",
```

Comment:
Right. I wanted to keep `@deprecated` method calls in tests which wouldn't compile with this flag. Ultimately I resigned from this idea so I can bring it back, thanks for pointing this out.